addon/lib/recall.js

```diff
@@ -6,12 +6,16 @@ const ss = require("sdk/simple-storage");
 const clipboard = require("sdk/clipboard");
 const { AbstractShot } = require("./shared/shot");
 
+let showInfoPanel = null;
```

Comment:
Yes, I thought I would keep it there just for completeness because I injected hideInfoPanel too, but I'll take it out.